{
  "description": "It was discovered that PySAML2 incorrectly handled certain\nSAML XML requests and responses. A remote attacker could use\nthis issue to read arbitrary files.\n",
  "releases": {
    "zesty": {
      "sources": {
        "python-pysaml2": {
          "version": "3.0.0-3ubuntu1.17.04.1",
          "description": "Pure python implementation of SAML2"
        }
      },
      "binaries": {
        "python-pysaml2": {
          "version": "3.0.0-3ubuntu1.17.04.1"
        },
        "python3-pysaml2": {
          "version": "3.0.0-3ubuntu1.17.04.1"
        }
      },
      "archs": {
        "source": {
          "urls": {
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.17.04.1.dsc": {
              "md5": "1adcdfee5cf4a20daca53a3cefd566e6",
              "size": 2807
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0.orig.tar.xz": {
              "md5": "99d7caa1a52ace54f33eb66d7b337df7",
              "size": 2697844
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.17.04.1.debian.tar.xz": {
              "md5": "53ab7b76ce318d8856a30835da127b31",
              "size": 8832
            }
          }
        },
        "all": {
          "urls": {
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2-doc_3.0.0-3ubuntu1.17.04.1_all.deb": {
              "md5": "0cf3fa0628c4d91d6efdab97e109d728",
              "size": 45654
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.17.04.1_all.deb": {
              "md5": "e6d7bf6269b309016ce292f8a93340be",
              "size": 195300
            },
            "http://security.ubuntu.com/ubuntu/pool/universe/p/python-pysaml2/python3-pysaml2_3.0.0-3ubuntu1.17.04.1_all.deb": {
              "md5": "6582488cbd8b1fb2fd4ff6775619dad2",
              "size": 195384
            }
          }
        }
      }
    },
    "xenial": {
      "allbinaries": {
        "python-pysaml2-doc": {
          "pocket": "security",
          "version": "3.0.0-3ubuntu1.16.04.1"
        },
        "python-pysaml2": {
          "pocket": "security",
          "version": "3.0.0-3ubuntu1.16.04.1"
        },
        "python3-pysaml2": {
          "pocket": "security",
          "version": "3.0.0-3ubuntu1.16.04.1"
        }
      },
      "binaries": {
        "python-pysaml2": {
          "pocket": "security",
          "version": "3.0.0-3ubuntu1.16.04.1"
        },
        "python3-pysaml2": {
          "pocket": "security",
          "version": "3.0.0-3ubuntu1.16.04.1"
        }
      },
      "archs": {
        "source": {
          "urls": {
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.16.04.1.debian.tar.xz": {
              "md5": "d729fb7974cdc77dd85565e5cec9b431",
              "size": 8836
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0.orig.tar.xz": {
              "md5": "99d7caa1a52ace54f33eb66d7b337df7",
              "size": 2697844
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.16.04.1.dsc": {
              "md5": "3f56515deb46aae03b52ff83519cb182",
              "size": 2807
            }
          }
        },
        "all": {
          "urls": {
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2_3.0.0-3ubuntu1.16.04.1_all.deb": {
              "md5": "95e87d4eaf1044e17beb8d50529ecd15",
              "size": 195692
            },
            "http://security.ubuntu.com/ubuntu/pool/main/p/python-pysaml2/python-pysaml2-doc_3.0.0-3ubuntu1.16.04.1_all.deb": {
              "md5": "27650c8ba8397c8ad4781cf4d0dff7ca",
              "size": 46658
            },
            "http://security.ubuntu.com/ubuntu/pool/universe/p/python-pysaml2/python3-pysaml2_3.0.0-3ubuntu1.16.04.1_all.deb": {
              "md5": "7250c01660a9c897719246001c991f43",
              "size": 195912
            }
          }
        }
      },
      "sources": {
        "python-pysaml2": {
          "version": "3.0.0-3ubuntu1.16.04.1",
          "description": "Pure python implementation of SAML2"
        }
      }
    }
  },
  "title": "PySAML2 vulnerability",
  "timestamp": 1503576131.945077,
  "summary": "python-pysaml2 vulnerability",
  "action": "In general, a standard system update will make all the necessary changes.\n",
  "isummary": "The system could be made to expose sensitive information.\n",
  "id": "USN-3402-1",
  "cves": [
    "CVE-2016-10149"
  ]
}